docker/hooks/build

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
```

Comment:
Please add license header here (as in twin https://github.com/telefonicaid/iotagent-json/pull/390/files#diff-914aa1982dfef1acb2b6267494d8895f)

Reply:
Fixed e23d56e